integration_tests/test_math.py

```diff
@@ -72,13 +72,22 @@ def test_ldexp():
 
 
 def test_fabs():
-    i: f64
-    j: f64
+
+
+    # i: f64
+    # j: f64
+    # i = fabs(10.3)
+    # j = fabs(-10.3)
+    # assert abs(i - j) < eps
     eps: f64
     eps = 1e-12
-    i = fabs(10.3)
-    j = fabs(-10.3)
-    assert abs(i - j) < eps
+
+    assert abs(fabs(10) - fabs(-10)) < eps
+    assert abs(fabs(10.3) - fabs(-10.3)) < eps
+    assert fabs(0.5) == 0.5
+    assert fabs(-5) == 5
```

Comment:
I see only constant values being passed as inputs. Can we pass things like variables, a complicated expression of variables, expressions like, `fabs(fabs(a + fabs(b)))`?

Reply:
Lets try this too!